lib/action.js

```diff
@@ -72,7 +73,8 @@ switch(query) {
       .catch(console.log);
   break;
   case 'transition':
-    let [ticketId, action, token] = args;
+    let [ticketId, action] = args;
+    token = keychain.find()
     Jira.transition(ticketId, action, token);
```

Comment:
According to the article linked above, it doesn't seem like the `atl_token` is even necessary anymore. I can push out a fix for this in an upcoming release once I have more time to research/test but in the meantime you can bypass it by simply passing a non-empty string to the method:
```
Jira.transition(ticketId, action, ' ');
```